contracts/linear/src/epoch_actions.rs

```diff
@@ -90,9 +90,10 @@ impl LiquidStakingContract {
             return false;
         }
 
-        let candidate = self
-            .validator_pool
-            .get_candidate_to_unstake(self.unstake_amount_to_settle, self.total_staked_near_amount);
```

Comment:
Should we remove the old `get_candidate_to_unstake` function as it's not needed any more?

Reply:
I think we'd better keep the old version for quick switching when problems arise with the new version

Reply:
sure. we can refactor code after the feature is stable.